doc/HOSTING

```diff
@@ -0,0 +1,116 @@
+# introduction
+
+This document tries to explain what is required to host a
+dedicated server, provided you already have a working
+unvanquished installation and "pak" files (those who have a
+`.dpk` file extension), which are not necessarily the
+official ones.
+
+# non-goals
+It does *not* tries to explain how to compile or modify
+unvanquised, how to configure a server with modified engine,
+how to setup your system or network, and aims at the
+simplest possible configuration.
+
+# pre-requisites
+
+First of all, you need a computer than can be reached by the
+Internet. A computer on your LAN might do the job, but you
+will need to configure your router beforehand.
+You will also need an HTTP server serving on port 80. While
+this is not strictly necessary, it is highly recommended.
+Not doing this means that if your players lack one of the
+requirements, they might be unable to join at all.
+I'll skip the details on this, as this discussion is out
+of the scope for this current document.
+
+# Daemon installation
+
+Create a directory in which you will place the following
+files:
+
+* daemonded
+* crash_server
+* nacl_helper_bootstrap
+* nacl_loader
+* irt_core-x86_64.nexe
+
+# HTTP configuration
+
+You will need to have a directory (I advise you to dedicate
+it to unvanquished, but this is not mandatory) containing
+`.dpk` files and another file named `PAKSERVER`, containing
+the string `ALLOW_UNRESTRICTED_DOWNLOAD`.
+If you get this wrong, your users will experience the
+infamous "UDP download", which is very slow. Slow as in,
```

Comment:
"which will net users 3 KiB/s".

All the follow up text doesn't really add anything to the documentation.

Reply:
It's basically to put emphasis on the fact it's a bad idea. But I guess you're right.